docker-compose.yaml

```diff
@@ -47,9 +56,17 @@ services:
       CONNECTOR_NODE_METADATA_URL: http://stub-connector/Metadata
       CONNECTOR_NODE_ENTITY_ID: http://localhost:5000/Metadata
       REDIS_SERVER_URI: redis://gateway-redis:6379/
+      SOFT_HSM_LIB_PATH: /usr/lib/softhsm/libsofthsm2.so
```

Comment:
Is this likely to change?, Can we put it in the docker file if it will stay the same 🙂